airflow/www/views.py

```diff
@@ -2288,6 +2291,38 @@ class UserModelView(wwwutils.SuperUserMixin, AirflowModelView):
     column_default_sort = 'username'
 
 
+class VersionView(wwwutils.SuperUserMixin, BaseView):
+    @expose('/')
+    def version(self):
+        # Look at the version from setup.py
+        airflow_version = None
+        try:
+            airflow_version = pkg_resources.require("airflow")[0].version
+        except :
+            pass
+
+        # Get the Git repo and git hash : first try pip freeze, then look at git
+        git_proj_uri = git_repo = git_hash = None
+        # Try looking at Git
+        try:
+            repo = Repo(os.path.expanduser(settings.AIRFLOW_HOME))
+            head_commit = repo.head.commit
+            git_hash = head_commit.hexsha
+            git_repo=repo.remotes.origin.url
```

Comment:
I played with this lib a bit just now in ipython and `repo.remotes.origin` didn't work for me.